src/magnify/apps/core/models.py

```diff
@@ -189,13 +212,66 @@ def __str__(self):
         return self.name
 
 
+class MeetingUserRelation(BaseModel):
```

Comment:
@kernicPanel I have renamed my models as per your suggestion. 
As for what I suggested for the API, I think it can be done later because I still have many endpoints to add before next week and I'm not sure how to do it correctly so will take time. It's easy to change it later so let's move on?

Reply:
> Yeah you're right thinking actually it should be an endpoint `/api/meeting/{id}/users` right?

Indeed, this is more REST :+1:

Reply:
> 



> @kernicPanel I have renamed my models as per your suggestion. As for what I suggested for the API, I think it can be done later because I still have many endpoints to add before next week and I'm not sure how to do it correctly so will take time. It's easy to change it later so let's move on for now?

As long as the models names won't change, I agree, yes.
Let's go to the next priority :rocket:

Reply:
Resolving the issue. We agreed IRL to refactor the endpoint in another PR handled by @kernicPanel :tada: